socs/agents/hwp_pmx/agent.py

```diff
@@ -212,7 +256,27 @@ def set_v(self, session, params):
         return True, 'Set voltage is done'
 
     @defer.inlineCallbacks
-    @ocs_agent.param('curr', default=1., type=float, check=lambda x: 0. <= x <= 3.)
+    def check_i_lim(self, session, params):
+        """check_i_lim()
+        **Task** - Check the current protection limit.
+        """
+        action = Actions.CheckILim(**params)
+        self.action_queue.put(action)
+        session.data = yield action.deferred
+        return True, 'Check current protection limit is done'
+
+    @defer.inlineCallbacks
+    def check_v_lim(self, session, params):
+        """check_v_lim()
+        **Task** - Check the voltage protection limit.
+        """
+        action = Actions.CheckVLim(**params)
+        self.action_queue.put(action)
+        session.data = yield action.deferred
+        return True, 'Check voltage protection limit is done'
+
+    @defer.inlineCallbacks
+    @ocs_agent.param('curr', default=1.3, type=float)
```

Comment:
Default here differs from the docstring of `curr=1`. Assuming 1.3 is the correct value, please update the docstring on line 281.